src/interfaces.ts

```diff
@@ -123,3 +123,19 @@ export interface ITestCase {
   new_name: string;
   testFn: Function;
 }
+
```

Comment:
ok i see where you're going with this. since stubs and mocks are released yet, we may need to have a discussion about how it all works--syntax being considered first though. do you mind dropping in the drash discord channel and talking about it in the dev channel?